app/uk/gov/hmrc/tradergoodsprofilesrouter/service/RequestAdviceService.scala

```diff
@@ -112,7 +112,7 @@ class RequestAdviceService @Inject() (
       goodsItemRecords.goodsDescription,
       Some(goodsItemRecords.countryOfOrigin),
       goodsItemRecords.supplementaryUnit,
-      Some(goodsItemRecords.category),
+      goodsItemRecords.category,
```

Comment:
This shouldn't be `Some(goodsItemRecords.category)` since category is optional?

Reply:
goodsItemRecord.category return an Option